CONTRIBUTING.md

```diff
@@ -1,14 +1,11 @@
-# Guidance on how to contribute
-
 > All contributions to this project will be released under the CC0 public domain
 > dedication. By submitting a pull request or filing a bug, issue, or
 > feature request, you are agreeing to comply with this waiver of copyright interest.
-> Details can be found in our [TERMS](TERMS.md) and [LICENSE](LICENSE).
-
+> Details can be found in our [TERMS](https://github.com/cfpb/capital-framework/blob/master/TERMS.md) and [LICENSE](https://github.com/cfpb/capital-framework/blob/master/LICENSE).
```

Comment:
We should stop abusing the `blockquote` formatting here. On the docs site, it just indents the paragraph, which looks odd. I'd say just let it be a regular paragraph and maybe put an `hr` underneath to separate the more practical section below.

Reply:
https://github.com/cfpb/capital-framework/pull/910/commits/010a88fe075bbee0e6e5c425d28f49551dc90744